### ISSUE TITLE
[Bug]: Cooldown Tracker resets when targeting a party member

### ISSUE THREAD

**Issue by kensykora** [DelvUI/DelvUI#1237]
### What are you trying to do?

When tracking cooldowns, clicking on another party member seems to reset both active cooldown durations remaining, and time until cooldown available timers. Everything resets to "Available" as soon as I target another party member.

This means that when I need to target folks to give them heart of corundum, i lose all my CD tracking. This will make raid callouts harder. It will eventually reset but all current timers are lost.

### What is the expected behavior?

When targeting another party member, current active cooldowns and unavailable CD timers continue tracking

### What actually happened?

As soon as I click on another party member, it resets

### Logs

```shell
Video capture demonstrating issue - https://streamable.com/hkb98c
```

### FFXIV Update

- [X] I have confirmed that I have the latest version of XIVLauncher and DelvUI.

**Comment by Zeffuro** [comment]
I can somewhat replicate this issue outside of a Duty too by going to Summerford Farms, summon Chocobo, attack dummy and use a raid buff, dismiss Chocobo while still in combat.
It could be that the issues are coming from some party member index/memory reading going wrong.

**Comment by Zeffuro** [comment]
Switching Party Members triggers

```
private void OnCooldownsChanged(PartyCooldownsManager sender)
        {
            UpdateCooldowns();
        }
```

**Comment by Zeffuro** [comment]
The problem lies here: https://github.com/DelvUI/DelvUI/blob/d58662e03a66130e8bba25546b478dc07e8e6c08/DelvUI/Interface/Party/PartyManager.cs#L470

When player targets another party member the old party information won't match the new one.
It's highly likely the issue lies in the offsets for PartyListMemberRawInfo: https://github.com/DelvUI/DelvUI/blob/d58662e03a66130e8bba25546b478dc07e8e6c08/DelvUI/Interface/Party/PartyManager.cs#L818

Here is sample data with the names changes, the ObjectId and Type gets changed when you target a party member and then boss again. Curiously this usually only occurs when nothing has gone on cooldown yet:

```
19:45:45.354 | INF | [DelvUI] New Party Members: Name: Player One, ObjectId: 2179858688, Type: 177, JobId: 40, Status: , Name: Player Two, ObjectId: 13561, Type: 0, JobId: 21, Status: In Duty, Name: Player Three, ObjectId: 13535, Type: 0, JobId: 32, Status: In Duty, Name: Player Four, ObjectId: 4294967295, Type: 8, JobId: 40, Status: In Duty, Name: Player Five, ObjectId: 1276386581, Type: 23, JobId: 22, Status: In Duty, Name: Player Six, ObjectId: 1260199697, Type: 23, JobId: 39, Status: In Duty, Name: Player Seven, ObjectId: 1270930054, Type: 23, JobId: 25, Status: In Duty, Name: Player Eight, ObjectId: 1272185933, Type: 23, JobId: 35, Status: In Duty | Old Party Members: Name: Player One, ObjectId: 2196635904, Type: 145, JobId: 40, Status: , Name: Player Two, ObjectId: 13561, Type: 0, JobId: 21, Status: In Duty, Name: Player Three, ObjectId: 13535, Type: 0, JobId: 32, Status: In Duty, Name: Player Four, ObjectId: 4294967295, Type: 8, JobId: 40, Status: In Duty, Name: Player Five, ObjectId: 1276386581, Type: 23, JobId: 22, Status: In Duty, Name: Player Six, ObjectId: 1260199697, Type: 23, JobId: 39, Status: In Duty, Name: Player Seven, ObjectId: 1270930054, Type: 23, JobId: 25, Status: In Duty, Name: Player Eight, ObjectId: 1272185933, Type: 23, JobId: 35, Status: In Duty
19:45:46.625 | INF | [DelvUI] New Party Members: Name: Player One, ObjectId: 2196635904, Type: 145, JobId: 40, Status: , Name: Player Two, ObjectId: 13561, Type: 0, JobId: 21, Status: In Duty, Name: Player Three, ObjectId: 13535, Type: 0, JobId: 32, Status: In Duty, Name: Player Four, ObjectId: 4294967295, Type: 8, JobId: 40, Status: In Duty, Name: Player Five, ObjectId: 1276386581, Type: 23, JobId: 22, Status: In Duty, Name: Player Six, ObjectId: 1260199697, Type: 23, JobId: 39, Status: In Duty, Name: Player Seven, ObjectId: 1270930054, Type: 23, JobId: 25, Status: In Duty, Name: Player Eight, ObjectId: 1272185933, Type: 23, JobId: 35, Status: In Duty | Old Party Members: Name: Player One, ObjectId: 2179858688, Type: 177, JobId: 40, Status: , Name: Player Two, ObjectId: 13561, Type: 0, JobId: 21, Status: In Duty, Name: Player Three, ObjectId: 13535, Type: 0, JobId: 32, Status: In Duty, Name: Player Four, ObjectId: 4294967295, Type: 8, JobId: 40, Status: In Duty, Name: Player Five, ObjectId: 1276386581, Type: 23, JobId: 22, Status: In Duty, Name: Player Six, ObjectId: 1260199697, Type: 23, JobId: 39, Status: In Duty, Name: Player Seven, ObjectId: 1270930054, Type: 23, JobId: 25, Status: In Duty, Name: Player Eight, ObjectId: 1272185933, Type: 23, JobId: 35, Status: In Duty
```

I'm not smart enough to fix this problem myself because I'm not wellversed in memory reading, especially not when I have to do/test this live in random trials (queueing as Sage in Castrum Fluminis works well though).

Since Savage release is upon us don't expect a fix for a good while.

**Comment by Tischel** [comment]
Fixed in v2.2.0.9
